frontend/src/organisms/Navigation.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { ReactComponent as Logo } from '../assets/images/fitify.svg';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { BurgerUser } from './BurgerUser';
+
+import { Link, NavLink } from 'src/atoms/';
+import { useAuth } from 'src/utils/auth';
+import { route } from 'src/Routes';
+import { faUserCircle } from '@fortawesome/free-solid-svg-icons';
+
+export function Navigation() {
+    const { user } = useAuth();
+    const [open, setOpen] = useState(false);
+
+    return (
+        <main className="navigation">
+            <nav className="flex justify-between bb bg-dark">
+                <Link to={route.home()} noUnderline className="b flex items-center green">
+                    <Logo className="mr1 f2 fitify-logo" />
+                </Link>
+                <div className="flex-grow flex items-center mr4">
+                    {user ? (
+                        <>
+                            <NavLink exact to={route.dashboard()} className="pa3 dib-ns">
+                                Dashboard
+                            </NavLink>
+                            <NavLink exact to={route.newTraining()} className="pa3 dib-ns">
+                                New Training
+                            </NavLink>
+
+                            <FontAwesomeIcon icon={faUserCircle} className={'ma2 green f2'}
+                                onClick={() => setOpen(!open)} />
+                                {open && <BurgerUser />}  
```

Comment:
proč v desktopové navigaci potřebuješ `BurgerUser` který ti ukazuje usera  logout?

Reply:
abych mohla pracovat s dashboardem a new training vlevo při větší obrazovce. Asi to půjde udělat i jinak, jen zatím nevím jak.